app/test_main.py

```diff
@@ -1,33 +1,57 @@
+import pytest
 from app.main import get_human_age
 
 
-def test_age_cant_be_zero() -> None:
-    assert get_human_age(0, 0) == [0, 0]
-
-
-def test_age_of_animals_less_15() -> None:
-    assert get_human_age(12, 14) == [0, 0]
-
-
-def test_animals_15_years_old() -> None:
-    assert get_human_age(15, 15) == [1, 1]
-
-
-def test_animals_23_years_old() -> None:
-    assert get_human_age(23, 23) == [1, 1]
-
-
-def test_animals_24_years_old() -> None:
-    assert get_human_age(24, 24) == [2, 2]
-
-
-def test_animals_27_years_old() -> None:
-    assert get_human_age(27, 27) == [2, 2]
-
-
-def test_animals_28_years_old() -> None:
-    assert get_human_age(28, 28) == [3, 2]
-
-
-def test_animals_100_years_old() -> None:
-    assert get_human_age(100, 100) == [21, 17]
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_years",
+    [
+        (
+            0, 0, [0, 0]
+        ),
+        (
+            14, 14, [0, 0]
+        ),
+        (
+            15, 15, [1, 1]
+        ),
+        (
+            23, 23, [1, 1]
+        ),
+        (
+            24, 24, [2, 2]
+        ),
+        (
+            27, 27, [2, 2]
+        ),
+        (
+            28, 28, [3, 2]
+        ),
+        (
+            100, 100, [21, 17]
+        )
+    ]
+)
+def test_get_correct_human_age(cat_age: int,
+                               dog_age: int,
+                               expected_years: list) -> None:
+    assert get_human_age(cat_age, dog_age) == expected_years
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_years",
+    [
+        (
+            -1, 8, [0, 0]
+        ),
+        (
+            2, -10, [0, 0]
+        ),
+        (
+            -2, -25, [0, 0]
+        )
+    ]
+)
+def test_negative_age(cat_age: int,
+                      dog_age: int,
+                      expected_years: list) -> None:
+    assert get_human_age(cat_age, dog_age) == expected_years
```

Comment:
i didn't add test for float numbers, because we have type annotation

Reply:
yeah, but type annotations do not secure your function from passing wrong arguments. You should test behavior of your function if wrong data type is passing to your function